www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogHandle.xml

```diff
@@ -1,34 +1,34 @@
 <form>
     <field>
         <id>handle.enabled</id>
-        <label>enabled</label>
+        <label>Enabled</label>
         <type>checkbox</type>
-        <help><![CDATA[Enable this handler.]]></help>
+        <help><![CDATA[Enable or disable this handler.]]></help>
```

Comment:
Fair point, but a bit spurious? :)

Reply:
Maybe I overdid it with how precise the help text is.